src/Tasks/WriteCodeFragment.cs

```diff
@@ -449,7 +449,7 @@ private bool AddArguments(
                             value = ConvertParameterValueToInferredType(
                                 constructorParameterTypes[i],
                                 parameter.Value,
-                                $"#{i + 1}"); /* back to 1 based */
+                                parameter.PositionalParameterName); /* back to 1 based */
```

Comment:
I believe that the `/* back to 1 based */` comment is obsolete due to this change.
```suggestion
                                parameter.PositionalParameterName); 
```